resources/views/detail/clan.blade.php

```diff
@@ -0,0 +1,85 @@
+@extends('layouts.modal')
+@isset($clan)
+    @section('modal-title', $clan["name"])
+@section('modal-content')
+    {{--    TODO Update HTML and translate messages--}}
+    <table width="100%">
+        <tbody>
+        <tr>
+            <td valign="top">
+                <h3>Members</h3>
+                <table class="table table-striped">
+                    <tbody>
+                    @foreach($clan["members"] as $member)
+                        <tr>
+                            <td><img src="https://mc-heads.net/avatar/{{ $member }}/16/"
```

Comment:
As I may remember, you may do this:
`{{ secure_url("mc-heads.net/avatar/$member") }}`

Reply:
That is for internal URLs I believe.